simulation-backend/simulation_backend/headless_backend/HelperFunctions.py

```diff
@@ -0,0 +1,66 @@
+import os
+import json
+
+
+def find_input_file_in_subfolders(
+    dirname=os.path.dirname(__file__), file_name="MeasurementRoom.json"
+):
+
+    for subdir, dirs, files in os.walk(dirname):
+        for file in files:
+            # print os.path.join(subdir, file)
+            filepath = subdir + os.sep + file
+
+            if filepath.endswith(file_name):
+                print(f"{file_name} has been found")
+                return filepath
+
+    print(f"{file_name} has NOT been found")
+    return ""
```

Comment:
I'm not sure how these things are handled in the rest of the backend code.
In general I would think that explicitly raising an exception makes sense here if the file is not found. This way things cannot silently pass.
See https://realpython.com/python-exceptions/

```suggestion
    raise FileNotFoundError(f"{file_name} has NOT been found")
```